oak_functions/util/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "oak_functions_util"
+version = "0.1.0"
+edition = "2021"
+license = "Apache-2.0"
+
+[dependencies]
+lock_api = { version = "*", features = ["arc_lock"] }
```

Comment:
The arc_lock feature requires the alloc crate to be available, so you should add `extern crate alloc;` to `lib.rs`. That should fix most of the build errors.

Reply:
Done.